iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationOSMLoginViewController.mm

```diff
@@ -138,6 +139,9 @@ - (IBAction)login
       catch (exception const & ex)
       {
         LOG(LWARNING, ("Error login", ex.what()));
+        [Statistics logEvent:@"Editor_Reg_request_result" withParameters:@{kStatIsSuccess : kStatNo,
```

Comment:
Представь, что есть две кнопки:
1. Регистрация.
2. Авторизация.

А по клику они вызывают один и тот же код.

Вот именно это у нас и происходит.

> On Mar 25, 2016, at 22:51, Dmitry Yunitsky <notifications@github.com> wrote:
> 
> In iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationOSMLoginViewController.mm <https://github.com/mapsme/omim/pull/2511#discussion_r57462735>:
> 
> > @@ -138,6 +139,9 @@ - (IBAction)login
> >        catch (exception const & ex)
> >        {
> >          LOG(LWARNING, ("Error login", ex.what()));
> > +        [Statistics logEvent:@"Editor_Reg_request_result" withParameters:@{kStatIsSuccess : kStatNo,
> Что значит идентичные? Регистрация - регистрирует нового ОСМ юзера, авторизация - авторизует старого, существующего. По-моему разница более чем явная.
> … <x-msg://39/#>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> 
>  <https://github.com/mapsme/omim/pull/2511#discussion_r57462415> <https://github.com/mapsme/omim/pull/2511#discussion_r57461965> <https://github.com/mapsme/omim/pull/2511#discussion_r57461714> <https://github.com/mapsme/omim/pull/2511#discussion_r57461616> <https://github.com/mapsme/omim/pull/2511#discussion_r57457291> <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57457291> <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57461616> <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57461714> <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57461965> <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57462415>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57462735>